example_library/bs-config.json

```diff
@@ -5,7 +5,7 @@
         "routes": {
             "/jsonforms.js": "dist/jsonforms.js",
             "/jsonforms.js.map": "dist/jsonforms.js.map",
-            "/native-shim.js": "example/native-shim.js",
```

Comment:
The example_library script isn't working for me (this might have been the case before). First, I think the `lite-server` devDependency is missing in `package.json`. Once installed, I get an error: 

```
Uncaught TypeError: Cannot read property 'prototype' of undefined
    at (index):25
``` 
which is this line:

```
Object.setPrototypeOf(CustomElement.prototype, JSONForms.Renderer.prototype);
```